src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -205,13 +215,62 @@ class CompanyStep extends React.Component {
+                            />
+                            <TextLink
+                                style={[styles.textMicro]}
+                                onPress={() => this.setState({manualAddress: true})}
+                            >
+                                Can&apos;t find your address? Enter it manually
+                            </TextLink>
+                        </div>
+                    )}
+                    {this.state.manualAddress && (
+                        <div>
+                            <ExpensiTextInput
+                                label={this.props.translate('common.companyAddress')}
+                                containerStyles={[styles.mt4]}
+                                onChangeText={value => this.clearErrorAndSetValue('addressStreet', value)}
+                                value={this.state.addressStreet}
+                                errorText={this.getErrorText('addressStreet')}
+                            />
+                            <Text style={[styles.mutedTextLabel, styles.mt1]}>{this.props.translate('common.noPO')}</Text>
+                            <View style={[styles.flexRow, styles.mt4]}>
+                                <View style={[styles.flex2, styles.mr2]}>
+                                    <ExpensiTextInput
+                                        label={this.props.translate('common.city')}
+                                        onChangeText={value => this.clearErrorAndSetValue('addressCity', value)}
+                                        value={this.state.addressCity}
+                                        errorText={this.getErrorText('addressCity')}
+                                        translateX={-14}
+                                    />
+                                </View>
+                                <View style={[styles.flex1]}>
+                                    <StatePicker
+                                        onChange={value => this.clearErrorAndSetValue('addressState', value)}
+                                        value={this.state.addressState}
+                                        hasError={this.getErrors().addressState}
+                                    />
+                                </View>
+                            </View>
+                            <ExpensiTextInput
+                                label={this.props.translate('common.zip')}
+                                containerStyles={[styles.mt4]}
+                                keyboardType={CONST.KEYBOARD_TYPE.PHONE_PAD}
+                                onChangeText={value => this.clearErrorAndSetValue('addressZipCode', value)}
+                                value={this.state.addressZipCode}
+                                errorText={this.getErrorText('addressZipCode')}
+                            />
+                        </div>
+                    )}
```

Comment:
Any specific reason we're not using a ternary here?

```suggestion
                    {!this.state.manualAddress ? (
                        <div>
                            <AddressSearch
                                label={this.props.translate('common.companyAddress')}
                                containerStyles={[styles.mt4]}
                                value={this.getFormattedAddressValue()}
                                onChangeText={(fieldName, value) => this.clearErrorAndSetValue(fieldName, value)}
                                errorText={this.getErrorText('addressStreet')}
                            />
                            <TextLink
                                style={[styles.textMicro]}
                                onPress={() => this.setState({manualAddress: true})}
                            >
                                Can&apos;t find your address? Enter it manually
                            </TextLink>
                        </div>
                    ) : (
                        <div>
                            <ExpensiTextInput
                                label={this.props.translate('common.companyAddress')}
                                containerStyles={[styles.mt4]}
                                onChangeText={value => this.clearErrorAndSetValue('addressStreet', value)}
                                value={this.state.addressStreet}
                                errorText={this.getErrorText('addressStreet')}
                            />
                            <Text style={[styles.mutedTextLabel, styles.mt1]}>{this.props.translate('common.noPO')}</Text>
                            <View style={[styles.flexRow, styles.mt4]}>
                                <View style={[styles.flex2, styles.mr2]}>
                                    <ExpensiTextInput
                                        label={this.props.translate('common.city')}
                                        onChangeText={value => this.clearErrorAndSetValue('addressCity', value)}
                                        value={this.state.addressCity}
                                        errorText={this.getErrorText('addressCity')}
                                        translateX={-14}
                                    />
                                </View>
                                <View style={[styles.flex1]}>
                                    <StatePicker
                                        onChange={value => this.clearErrorAndSetValue('addressState', value)}
                                        value={this.state.addressState}
                                        hasError={this.getErrors().addressState}
                                    />
                                </View>
                            </View>
                            <ExpensiTextInput
                                label={this.props.translate('common.zip')}
                                containerStyles={[styles.mt4]}
                                keyboardType={CONST.KEYBOARD_TYPE.PHONE_PAD}
                                onChangeText={value => this.clearErrorAndSetValue('addressZipCode', value)}
                                value={this.state.addressZipCode}
                                errorText={this.getErrorText('addressZipCode')}
                            />
                        </div>
                    )}
```

Reply:
I'm not a huge fan of ternaries in this situation, and, at some point, I was told not to use them and do it like this instead 🤷